src/main/java/TaskManager.java

```diff
@@ -0,0 +1,126 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    protected ArrayList<Task> tasks;
+    protected DukeInterface dukeUI;
+    private int numTasks;
+    private int numComplete;
+
+    private final String ADD_TASK_VALID_MSG = "Chomp-chomp! I've added this new task [\uD83D\uDCDD]:";
+    private final String SET_TASK_COMPLETE_VALID_MSG = "Burrrp! I've marked this task as done [\u2705]:";
+    private final String SET_TASK_COMPLETE_INVALID_MSG = "Hold up! This task is already marked done [\u2705]:";
+    private final String PRINT_TASK_VALID_MSG = "Ahh! Here are the tasks in your list [\uD83D\uDCC5]:";
+    private final String PRINT_TASK_INVALID_MSG = "Sorry, but there are currently no tasks added!";
+    private final String GENERIC_PROMPT_MSG = "You may choose to add/complete another task.";
+
+    public TaskManager() {
+        tasks = new ArrayList<Task>();
+        dukeUI = new DukeInterface();
+        numTasks = 0;
+        numComplete = 0;
+    }
+
+    public void addTask(String[] taskInfo) {
```

Comment:
This method has more than 30 lines, maybe can adapt to follow the instructions for coding quality